client-src/src/app/app.html

```diff
@@ -17,7 +17,7 @@
 <ion-menu [content]="content" id="counselor">
   <ion-header>
     <ion-toolbar>
-      <ion-title>Menu</ion-title>
+      <ion-title>Welcome!</ion-title>
```

Comment:
Use Welcome {userName}

Reply:
Do I hard code the user name at the moment?